src/DeerSighting/deersightinglocation.tsx

```diff
@@ -0,0 +1,24 @@
+import { API_PATH } from '@/utils/constants';
+
+export interface DeerSighting {
+  longitude: number;
+  latitude: number;
+  timestamp: string;
+  image: null;
+}
```

Comment:
Create a `DeerSighting.ts` Interface file and move the interface code into that file. The interface file will be under the `interfaces/` folder